### ISSUE TITLE
Auth service unit test is flaky

### ISSUE THREAD

**Issue by garethbowen** [medic/cht-core#6439]
One of the Auth unit tests fails from time to time on travis.

```
1) splits comma separated permissions
     auth directive
     Error: Uncaught AssertionError: expected true to equal false (/home/travis/build/medic/medic/node_modules/chai/chai.js:239)
```

and

```
1) should be shown when auth does not error
     auth directive
     Error: Uncaught AssertionError: expected true to equal false (/home/travis/build/medic/cht-core/node_modules/chai/chai.js:239)
```

**Comment by garethbowen** [comment]
Added to 3.10.0 as this is failing quite a lot of our builds. Don't spend too long trying to fix it as we don't want to delay 3.10.0 - the best option may be deleting/disabling the tests altogether.

**Comment by garethbowen** [comment]
No AT required. Merged.
